src/System.Diagnostics.Tools/tests/System/Diagnostics/CodeAnalysis/SuppressMessageAttributeTests.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+namespace System.Diagnostics.CodeAnalysis.Tests
+{
+    public static class SuppressMessageAttributeTests
+    {
+        [Fact]
+        public static void TestConstructorBasic()
+        {
+            string expectedCategory = "Category";
+            string expectedCheckId = "CheckId";
+            SuppressMessageAttribute sma = new SuppressMessageAttribute(expectedCategory, expectedCheckId);
+            Assert.Equal(expectedCategory, sma.Category);
+            Assert.Equal(expectedCheckId, sma.CheckId);
+        }
```

Comment:
It'd be good to also test a few values known to sometimes cause exceptions (but shouldn't here), e.g.
```C#
[Theory]
[InlineData(null, "CheckId")]
[InlineData("", "CheckId")]
[InlineData("Category", null)]
[InlineData("Category", "")]
[InlineData("Category", "CheckId")]
public void TestConstructorBasic(string category, string checkId)
{
    var sma = new SuppressMessageAttribute(category, checkId);
    Assert.Equal(category, sma.Category);
    Assert.Equal(checkId, sma.CheckId);
}
```